lib/js/components/Modals/Modal/Modal.vue

```diff
@@ -264,53 +256,46 @@ $image-height-small: 140px;
 		}
 
 		&.-singleColumn {
-			#{$self}__footerColumn {
+			#{$self}__footerCtaPrimary {
 				justify-content: center;
 				width: 100%;
 			}
 		}
 	}
 
-	&__footerColumn {
+	&__tertiaryButton:only-child {
+		margin: 0 auto;
+	}
+
+	&__footerCtaPrimary {
 		display: flex;
 		flex-direction: column-reverse;
-		gap: 0 $space-s;
-
-		&.--cta {
-			@media #{breakpoint-s()} {
-				flex-direction: row;
-			}
+		gap: $space-s 0;
 
-			&:not(:first-child) {
-				// first-child because of reverse order in flex-direction
-				margin-bottom: $space-xs;
-
-				@media #{breakpoint-s()} {
-					margin-bottom: 0;
-				}
-			}
+		@media #{breakpoint-s()} {
+			flex-direction: row;
+			gap: 0 $space-s;
 		}
 
-		&.--ctaSecondary {
-			align-items: center;
-			gap: 0 $space-m;
-			justify-content: center;
+		&:not(:first-child) {
+			// first-child because of reverse order in flex-direction
+			margin-bottom: $space-xs;
 
 			@media #{breakpoint-s()} {
-				flex-direction: row;
-				justify-content: left;
+				margin-bottom: 0;
 			}
 		}
 	}
 
-	&__buttonPrimary {
-		&:not(:first-child) {
-			// first-child because of reverse order in flex-direction
-			margin-bottom: $space-s;
+	&__footerCtaSecondary {
+		align-items: center;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
 
-			@media #{breakpoint-s()} {
-				margin-bottom: 0;
-			}
+		@media #{breakpoint-s()} {
+			gap: 0 $space-m;
```

Comment:
don't we need to define this gap also for mobile, to reserve minimal spacing between checkbox and button when labels are long?

Reply:
[`c510547` (#278)](https://github.com/bethinkpl/design-system/pull/278/commits/c51054708fce3d4d40a8cd29120373183531e49a)